src/main/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiAction.java

```diff
@@ -72,10 +72,10 @@ public ActionGroupsApiAction(final Settings settings, final Path configPath, fin
 		super(settings, configPath, controller, client, adminDNs, cl, cs, principalExtractor, evaluator, threadPool, auditLog);
 	}
 
-	@Override
-	public List<Route> routes() {
-		return routes;
-	}
+    	@Override
```

Comment:
Either use tab or use spaces for indentation, mix and match is not an option. This file uses tabs for indentation.

Reply:
I believe I used tab to fix all the files with indentation issue.

Reply:
Please check again:
<img width="747" alt="Screen Shot 2021-09-14 at 13 13 19" src="https://user-images.githubusercontent.com/12200381/133327321-78a686ba-c068-4d2e-9e2e-84accafef85f.png">

Reply:
But in vim it does not show me any issue with indentation:
<img width="1767" alt="vim" src="https://user-images.githubusercontent.com/908933/133328588-4d394da7-c318-4d6f-9829-a5430d4e3153.png">
and on github:
<img width="934" alt="github" src="https://user-images.githubusercontent.com/908933/133329052-2cc2765f-f036-422b-9819-3862ed9ae2d0.png">

Reply:
set tab size to anything other than 8 and you will see the problem.

Reply:
Got it, fixed them all